src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs

```diff
@@ -1123,6 +1273,54 @@ SingleResult ToResult(TimeSeriesOperation.AppendOperation element)
             }
         }
 
+
+        private static readonly byte[] TimedCounterPrefixBuffer = Encoding.UTF8.GetBytes(TimedCounterPrefix);
+        private static readonly byte[] TimedCounterPositivePrefixBuffer = Encoding.UTF8.GetBytes(TimedCounterPositivePrefix);
+        private const string TimedCounterPrefix = "TC:";
+        private const string TimedCounterPositivePrefix = TimedCounterPrefix + "INC-";
+        private const string TimedCounterNegativePrefix = TimedCounterPrefix + "DEC-";
+        public string IncrementTimestamp(
+            DocumentsOperationContext context,
+            string documentId,
+            string collection,
+            string name,
+            IEnumerable<TimeSeriesOperation.IncrementOperation> toIncrement)
+        {
+#if DEBUG
+            DateTime prevTime = DateTime.MinValue;
+#endif
+
+            var holder = new SingleResult();
+
+            return AppendTimestamp(context, documentId, collection, name, toIncrement.Select(ToResult));
+
+
+            SingleResult ToResult(TimeSeriesOperation.IncrementOperation element)
+            {
+#if DEBUG
```

Comment:
Why debug? what would happened in release for such scenario?

Reply:
Done